eve/main.yml

```diff
@@ -30,7 +30,23 @@ stages:
         retryFetch: true
         haltOnFailure: true
     - ShellCommand:
-        name: 'Run test-suite'
+        name: 'Init node'
         haltOnFailure: true
         command: |-
-          bash ./vendor/bash_unit/bash_unit tests/single-node/test.sh
+          bash tests/single-node/init.sh
+    - ShellCommand:
+        name: 'Run the install test suite'
```

Comment:
Since we'll have several install scenarios (from scratch, upgrade, single-node, multi-node,...) could we already 'implement' this requirement, and run a single-node simple deployment twice?

Reply:
But we can have "from scratch single-node", "from scratch multi-node". My belief is that all permutation can't be handle in pytest. And then if we decide for a subset of all the possibility, this will prevent a user to come with it's own inventory and run the test-suite. And no user will ever run the test-suite again (Number of platform will consume all the scality.cloud quota required to run test-suite (and the insane time required to run multi-platform tests)), and non-scality member will struggle with integration with cloud provider (which is not nice for open-source project)

Reply:
OK